### PR TITLE
Update README.md and metadata.json (#7793)

### PR DESCRIPTION
**Pull request by HamzaBoukraa**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

[X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog
* Fixing errors and warnings of https://dev.azure.com/azurequickstarts/azure-quickstart-templates/_build/results?buildId=27562&view=results.

### CODE REVIEW

**Comment by HamzaBoukraa** [comment]
@bmoore-msft : 
Hello Brian,
I have split the changes into different PR as requested.

File: 101-cognitive-services-Computer-vision-API/metadata.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
   "itemDisplayName": "Cognitive Services Computer Vision API",
   "description": "A template for creating a new Cognitive Services Computer Vision API",
-  "summary": "A template for creating a new Cognitive Services Computer Vision API. Use this template as the framework for your custom deployment.",
-  "githubUsername": "olandese",
-  "dateUpdated": "2019-11-25",
+  "summary": "A template for creating a new Cognitive Services Computer Vision API. Use this template as the framework for your custom deployment. This template was originally created by Marco Mansi (@olandese).",
+  "githubUsername": "HamzaBoukraa",
```
Comment (bmoore-msft) [inline 101-cognitive-services-Computer-vision-API/metadata.json:6]: is your intent to take ownership of this sample?

Comment (HamzaBoukraa) [reply 101-cognitive-services-Computer-vision-API/metadata.json:6]: @bmoore-msft not at all, it was not my intention. I just was Fixing errors and warnings of https://dev.azure.com/azurequickstarts/azure-quickstart-templates/_build/results?buildId=27562&view=results

I thought the metadata is for this commited version : date + commiter.

Comment (HamzaBoukraa) [reply 101-cognitive-services-Computer-vision-API/metadata.json]: It clearly says that it is the original author , so I reverted the change related to githubUserName !
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md#githubusername

**Pull request merged**